api/models/endpoints.model.js

```diff
@@ -0,0 +1,7 @@
+const { readFile } = require("fs/promises");
+
+exports.selectEndpoints = () => {
```

Comment:
There is no need for a model for this endpoint. The model functionality is reserved purely for database interaction. You can move this code into the controller 👍